web-client/src/modules/requests/containers/RequestsContainer.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import PostsContainer from './PostsContainer';
+import { requestsPostType} from '../constants';
+
+const RequestsContainer: React.FC = () => (
```

Comment:
This container doesn't seem to have any logic in it, so why is it considered as a container?